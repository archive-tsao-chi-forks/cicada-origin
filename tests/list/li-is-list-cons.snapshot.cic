@def expectations: List(String) = @li [
  "cooked",
  "eaten",
  "tried-cleaning",
  "understood",
  "slept"
]

@show expectations

@show
@the List(String)
li("cooked",
  li("eaten",
    li("tried-cleaning",
      li("understood",
        li("slept",
          nil)))))
