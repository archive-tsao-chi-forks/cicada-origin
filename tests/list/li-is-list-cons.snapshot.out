@the List(String) li("cooked", li("eaten", li("tried-cleaning", li("understood", li("slept", nil)))))
@the List(String) li("cooked", li("eaten", li("tried-cleaning", li("understood", li("slept", nil)))))

